Sources/Commands/SwiftRunTool.swift

```diff
@@ -87,6 +87,17 @@ public class SwiftRunTool: SwiftTool<RunToolOptions> {
             print(Versioning.currentVersion.completeDisplayString)
 
         case .run:
+            // Detect deprecated uses of swift run to interpret scripts.
+            if let executable = options.executable, executable.lowercased().hasSuffix(".swift") {                
+                print("warning: 'swift run file.swift' command to interpret swift files is deprecated; " +
+                    "use 'swift file.swift' instead")
+                // Prepend the script to interpret to the arguments.
+                options.arguments.insert(executable, at: 0)
+                // Redirect execution to the toolchain's swift executable.
+                try run(getToolchain().swiftCompiler.appending(component: "..").appending(component: "swift"))
```

Comment:
We should put `swift` path in the toolchain. We can call it`swiftInterpreter`. Also, run method should take args array as an argument.